tasks/virtualenv.yml

```diff
@@ -31,6 +32,33 @@
   - name: cffi
   loop_control:
     loop_var: ler53_item
+  when: ansible_os_family == 'RedHat'
+  tags:
+  - install
+
+- name: Install virtualenv
+  pip:
+    name: virtualenv
+    state: present
+  when: ansible_os_family == 'Darwin'
+  tags:
+  - install
+
+- name: Install cryptography and boto3 in virtualenv
```

Comment:
Could you please explain why the existing task of `install pyOpenSSL and boto in a virtualenv` can't be reused?

Reply:
nice catch. 
I have no idea why I added a separate task for this. I can absolutely re-use the Red Hat one.